ocfweb/stats/printing.py

```diff
@@ -65,10 +66,7 @@ def _semester_histogram():
 
 @periodic(3600)
 def _toner_changes():
-    return [
-        (printer, _toner_changes_for_printer(printer))
-        for printer in ALL_PRINTERS
-    ]
+    return [(printer, _toner_used_by_printer(printer)) for printer in ACTIVE_PRINTERS]
```

Comment:
I think this looked better as a multi-liner.

Reply:
My bad, screenshot here: https://imgur.com/a/IeWHB

Reply:
For future reference, you can upload pictures to GitHub when you make a comment.